src/west/cmd/project.py

```diff
@@ -528,35 +527,35 @@ def _rebase(project):
     _git(project, 'rebase (manifest-rev-branch)')
 
 
-def _cloned(project):
-    # Returns True if the project's repository exists and looks like a Git
-    # repository.
+def _cloned(project, verify_repo=False):
```

Comment:
`verify_repo` never seems to be `True` now. Is that intentional?

Reply:
Looking at the callers, I believe we never want it set as the code stands:

- ListProjects definitely doesn't want it set
- It would be redundant in `Rebase`, `Branch`, `Checkout`, `Diff`, `Status`, `Forall`; since the `_projects` calls there leave `listed_must_be_cloned` its default, `True`, the `_cloned` calls are just filtering out cloned projects when the user doesn't list any
- The `listed_must_be_cloned` logic in `_projects` needs the complete list of uncloned projects, so it can't halt on the first one by setting it
- `_fetch` has logic for either return value, so it shouldn't be set there

Would you agree?

I've left it in to capture the results of our discussion for future use as needed. I'm happy to just remove it for YAGNI.

Reply:
The assumption before was that if a project's directory exists, then you want to check that it's a valid repository and report an error an error otherwise.

The new behavior is to ignore the project instead. I think we could go with that for now, and in that case, the `verify_repo` flag could be removed.

Reply:
> the verify_repo flag could be removed.

I've done so since it's unused code, despite not being sure I agree with your other point.